### PR TITLE
Fix/ie11 trouble with spinner branch

### PR DESCRIPTION
**Pull request by tilt**
@myabc Would you be so kind and try / merge it?

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -103,6 +103,7 @@ class Application < Rails::Application
     config.assets.paths << bower_assets_path.join(*%w(select2)).to_s
     config.assets.paths << bower_assets_path.join(*%w(jquery-ui themes base)).to_s
     config.assets.paths << bower_assets_path.join(*%w(jquery.atwho dist)).to_s
+    config.assets.paths << bower_assets_path.join(*%w(angular-busy dist)).to_s
```
Comment (myabc) [inline config/application.rb]: @tilt this line is not needed.

File: app/assets/stylesheets/_external.sass.erb [hunk 2]
```diff
@@ -31,3 +31,4 @@
 
 @import "<%= asset_path("select2/select2.css") %>"
 @import "<%= asset_path("jquery-ui/themes/base/jquery-ui.css") %>"
+@import "<%= asset_path("angular-busy/dist/angular-busy.css") %>"
```
Comment (myabc) [inline app/assets/stylesheets/_external.sass.erb:34]: @tilt you need to add this to the list in `production.rb`:

``` ruby
config.assets.precompile += %w(angular-busy/dist/angular-busy.css)
```

Comment (tilt) [inline app/assets/stylesheets/_external.sass.erb:34]: Thanks, done that

**Comment by ulferts** [comment]
Looks good on my machine. Merge once Travis agrees.

**Comment by tilt** [comment]
@ulferts Travis agreed, I just forgot to add [ci skip] to the asset path fix so go ahead :)

**Pull request merged**